resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.spec.erb

```diff
@@ -223,7 +223,13 @@ fi
 <% end -%>
 
 %post
+# Unconditional post-install tasks
 %{_app_prefix}/scripts/install.sh postinst_redhat
+
+# Only run these on initial install, not ugrades
+if [ "$1" = "1" ]; then
+  %{_app_prefix}/scripts/install.sh postinst_redhat_install
+fi
```

Comment:
With the FPM stuff you actually want to be updating this file, https://github.com/puppetlabs/ezbake/blob/master/resources/puppetlabs/lein-ezbake/template/global/ext/redhat/postinst.erb

The spec file is only used for legacy builds. I want to burn that down for exactly this reason Soon, but it's not there yet

![sad](https://media.giphy.com/media/AYKv7lXcZSJig/giphy.gif)

Reply:
oh cool, ok I was wondering how the standalone files were being used.